### PR TITLE
Small improvements

### PR DESCRIPTION
**Pull request by tomhukins**

### CODE REVIEW

File: lib/Class/MOP/Class.pm [hunk 1]
```diff
@@ -946,8 +946,7 @@ sub superclasses {
     my $isa = $self->get_or_add_package_symbol('@ISA');
 
     if (@_) {
-        my @supers = @_;
-        @{$isa} = @supers;
+        @{$isa} = @_;
```
Comment (autarch) [inline lib/Class/MOP/Class.pm]: I have to wonder if this code was the way it was because of some weird corner case with `@_`. Of course, there was no comment there and this isn't causing a test failure, so I'm probably being unnecessarily paranoid.

**Comment by tomhukins** [comment]
This pull request has been open for more than two years.  It fixes problems in Moose's documentation.  I welcome suggestions for what I might do to get these fixes merged.

**Comment by karenetheridge** [comment]
The process is to post a new comment, requesting that a maintainer take a look and merge it or comment on it. There's nothing more magical than that -- no one these days is going through all of the repositories they have merge access to and looking for things; they just need a poke.

Thank you for the reminder, and I will merge it today.

**Comment by karenetheridge** [comment]
I omitted the middle commit; the history might be revealing there but it is back in the Class-MOP repository, which I do not have archived.

**Comment by tomhukins** [comment]
Many thanks, @karenetheridge
